src/libcore/num/mod.rs

```diff
@@ -2037,6 +2037,115 @@ assert_eq!((-a).rem_euclid(-b), 1);
             }
         }
 
+        doc_comment! {
+            concat!("Returns the logarithm of the number with respect to an arbitrary base.
+
+Returns `None` if the number is lower than 1.
```

Comment:
"lower than 1" sounds unnecessarily circumspect to me. We're able to provide different wording for the signed and unsigned types so I think it would be clearer to say "zero" for the unsigned and "negative or zero" for the signed.